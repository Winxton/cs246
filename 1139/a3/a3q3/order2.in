! f
! e
! d
! c
! b
! a
? c
p
