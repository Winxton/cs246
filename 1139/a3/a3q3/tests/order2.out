a,b,c
a is parent of b
b is parent of c
c is parent of d
d is parent of e
e is parent of f
f has no children
